network-api/networkapi/campaign/views.py

```diff
@@ -194,6 +203,13 @@ def petition_submission(request, petition):
         }
     })
 
+    if settings.MOFO_NEWSLETTER_SUBSCRIBE_METHOD == 'BASKET':
```

Comment:
If I understood our conversation with Pmac correctly, we just want to subscribe them to the newlsetter through basket, and then send their petition information to SQS which is what this block of code doing.

@benhohner was wondering if you could please confirm that this is the desired flow :)

Reply:
Correct, this is the desired flow!